Cargo.toml

```diff
@@ -14,3 +14,37 @@ members = [
 	"test",
 	"ws",
 ]
+
+[workspace.package]
+version = "18.0.1"
+authors = ["Parity Technologies <admin@parity.io>"]
+repository = "https://github.com/paritytech/jsonrpc"
+homepage = "https://github.com/paritytech/jsonrpc"
+license = "MIT"
+edition = "2018"
+keywords = ["jsonrpc", "json-rpc", "json", "rpc", "serde"]
+
+[workspace.dependencies]
```

Comment:
Adding workspace dependencies and also keeping mentions to crates in the workspace in the workspace dependencies.